src/components/SelectionList/Search/ReportListItem.tsx

```diff
@@ -86,8 +86,11 @@ function ReportListItem<TItem extends ListItem>({
         return null;
     }
 
-    const participantFrom = reportItem.transactions[0].from;
-    const participantTo = reportItem.transactions[0].to;
+    // We get the from and to details from the first transaction. The first transaction's from and to need not be the same
+    // as the report's accountID and managerID.
+
+    const participantFrom = reportItem.transactions[0].from?.accountID === reportItem?.accountID ? reportItem.transactions[0].from : reportItem.transactions[0].to;
+    const participantTo = reportItem.transactions[0].to?.accountID === reportItem?.managerID ? reportItem.transactions[0].to : reportItem.transactions[0].from;
```

Comment:
I'm confused about why we have this check. Shouldn't we just use the reportItem from/to?

Reply:
`reportItem` only has account IDs. We need display name, avatar etc. which are available in `from` and `to` fields of transactions.

Reply:
Hmm can we update the reportItem to pass the personalDetails as well, like we do for transactions? I think that'd be cleaner and less confusing

Reply:
Yes. That will make the code on front-end cleaner.

Reply:
Cool, let's update it then!